src/Microsoft.Framework.Runtime.Interfaces/ILibraryManager.cs

```diff
@@ -6,6 +6,9 @@
 
 namespace Microsoft.Framework.Runtime
 {
+    /// <summary>
+    /// Provides access to the rich application dependency graph.
```

Comment:
"rich"? Is there a "poor" graph? :)

"complete" sounds a little better. Maybe even just rewrite it a bit:

> Provides access to the complete graph of dependencies for the application.